docs/manual_ES_model.md

```diff
@@ -0,0 +1,74 @@
+##Reference Manual
+### Elastic Search Model
+
+This module makes it possible to store, retrieve and search data from <a href = "https://www.elastic.co/downloads/elasticsearch">Elasticsearch </a>. It uses the <a href = "github.com/dhavalkapil/elasticsearch-lua">elasticsearch-lua</a> client for Lua. Refer the client <a href="http://elasticsearch-lua.readthedocs.io/en/latest/">here</a> to know the parameters which can be sent through the model functions if any. Return responses are also same as the one offered by this client. 
+
+After importing "es_model" we proceed to create a new instance of the model. There should be a file inside models/ which contains the keys and types defined for the particular model. 
+
+	local es_model = require "sailor.es_model"
+	local contact = contact.new("test")
```

Comment:
I addressed it and also added a commit. Please check https://github.com/sailorproject/sailor/pull/125/commits/2c8be478eca315fe7dc9c97898ef8d5df202ad11